repository/t.json

```diff
@@ -2897,6 +2897,19 @@
 				}
 			]
 		},
+		{
+			"name": "TokensCounter",
+			"details": "https://github.com/yaroslavyaroslav/SublimeTokenCounter",
+			"issues": "https://github.com/yaroslavyaroslav/SublimeTokenCounter/issues",
```

Comment:
That's the default URL for issues so no need to add it here, PackageControl will find that automatically.

```suggestion
```